backend/hitas/views/reports.py

```diff
@@ -124,3 +126,40 @@ def list(self, request: Request, *args, **kwargs) -> HttpResponse:
         workbook = build_multiple_ownerships_report_excel(ownerships)
         filename = "Useamman sääntelyn piirissä olevan asunnon omistavat omistajat.xlsx"
         return get_excel_response(filename=filename, excel=workbook)
+
+
+class OwnershipsByCompanyJSONReportView(ViewSet):
+    NON_DISCLOSURE_NAME = "xxx"
```

Comment:
Why is the 'obfuscated name' here `xxx` and in `OBFUSCATED_OWNER_NAME` it's `***`? 🤔

Reply:
Just to check if it is caught in the review! ;)